src/main/java/com/hyperwallet/clientsdk/util/HyperwalletApiClient.java

```diff
@@ -196,4 +210,23 @@ private String decryptResponse(String responseBody) throws ParseException, IOExc
         return isEncrypted ? hyperwalletEncryption.decrypt(responseBody) : responseBody;
     }
 
+    public WebResource getWebResource(final String url) {
+        client.addFilter(new HTTPBasicAuthFilter(this.username, this.password));
+        return client.resource(url);
+    }
+
+    public <T> T put(final String url, final FormDataMultiPart formDataMultiPart, final Class<T> type) {
+        Response response = new Response();
+        try {
+            webResource = getWebResource(url);
+            ClientResponse clientResponse = webResource.type(MediaType.MULTIPART_FORM_DATA_TYPE).put(ClientResponse.class, formDataMultiPart);
```

Comment:
I think we can have our own small implementation of pushing multipart http thru normal io streams, can we do that instead? rather than importing the whole jersey client?

Reply:
hw2 app is consuming the documents with FormDataMultiPart

    public Response uploadDocuments(@UserToken @PathParam("userToken") String userToken, final FormDataMultiPart multiPart,
            @Context UriInfo uriInfo)
            throws Exception {

That is the reason why we went with this approach. Please let me know if still we need to change it.